<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_16">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_24">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Password :</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLineEdit" name="LE_name">
           <property name="text">
            <string>ali</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit"/>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>@IP</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLineEdit" name="LE_ip">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="LE_port">
           <property name="text">
            <string>1213</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="BT_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Broadcast  To ALL Users :</string>
         </property>
        </widget>
        <widget class="QListWidget" name="list_broadcast"/>
        <widget class="QLineEdit" name="broadcast"/>
       </widget>
       <widget class="QSplitter" name="splitter_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Message  :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QListWidget" name="list_chat"/>
         </widget>
         <widget class="QSplitter" name="splitter_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Historique Message  :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QListWidget" name="list_hist"/>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="list_msg"/>
          <widget class="QLineEdit" name="message"/>
         </widget>
         <widget class="QSplitter" name="splitter_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Friend List :</string>
           </property>
          </widget>
          <widget class="QListWidget" name="list_client"/>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
